recipes/cuda-opencl/meta.yaml

```diff
@@ -0,0 +1,103 @@
+{% set name = "cuda-opencl" %}
+{% set version = "12.0.76" %}
+{% set lib_version = "1.0.0" %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "x64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_opencl/{{ platform }}/cuda_opencl-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 285b646a231d1101d7ffb7b37ca081f2c170114102ebfd0439fe189dbdc07b37  # [linux64]
+  sha256: 9a8688abf527dd1105e2f366e5aa84f901975a3ba94d977088764c45586370d3  # [win]
+
+
+build:
+  number: 0
+  skip: true  # [osx or aarch64 or ppc64le]
+
+test:
+  commands:
+    - test -f $PREFIX/targets/{{ target_name }}/lib/libOpenCL.so.{{ lib_version }}                                                # [linux]
+    - test -L $PREFIX/targets/{{ target_name }}/lib/libOpenCL.so.{{ lib_version.split(".")[0] }}                                  # [linux]
+    - test -L $PREFIX/targets/{{ target_name }}/lib/libOpenCL.so.{{ lib_version.split(".")[0] }}.{{ lib_version.split(".")[1] }}  # [linux]
```

Comment:
If we're doing Windows builds, we should have some kind of test for it.

Reply:
We now create `*.icd` files, which are tested on both platforms